WatchFaceAlphaKotlin/app/src/main/AndroidManifest.xml

```diff
@@ -78,7 +78,7 @@
         </service>
 
         <activity
-            android:name="androidx.wear.complications.ComplicationHelperActivity"
+            android:name="androidx.wear.watchface.ComplicationHelperActivity"
             android:exported="false" />
```

Comment:
I believe https://android-review.googlesource.com/c/platform/frameworks/support/+/1752949/ means that we can get rid of this declaration entirely, since it will automatically be added by the library's manifest.

Reply:
Sorry,  I forgot to merge this and I have a couple PRs that rely on this. I will fix in a follow up PR.